src/app/util/af-types.h

```diff
@@ -1977,6 +1977,8 @@ typedef struct
 #define EMBER_AF_PERMIT_JOIN_FOREVER 0xFF
 #define EMBER_AF_PERMIT_JOIN_MAX_TIMEOUT 0xFE
 
+#define MAX_INT16U_VALUE (0xFFFF)
```

Comment:
UINT16_MAX?

Reply:
Copying over what's in the silabs SDK for now.

Reply:
ack